xc/xenops_server_xen.ml

```diff
@@ -2160,6 +2160,35 @@ module VIF = struct
 			_locking_mode, "disabled";
 		]
 
+	let xenstore_of_static_ip_setting vif =
+		let constant_setting = [ "static-ip-setting/mac", vif.mac;
```

Comment:
It would be good to factor out some of these strings into as above for the locking mode ones.